src/Components/Comments.jsx

```diff
@@ -0,0 +1,40 @@
+import { React, useEffect, useState } from "react";
+import { getCommentsByArticleId } from "../utils/getCommentsByArticleId";
+import { useParams } from "react-router-dom";
+
+const Comments = () => {
+    const [isLoading, setIsLoading] = useState(true);
+    const [comments, setComments] = useState({})
+    const { article_id } = useParams();
+
+    useEffect(() => {
+        setIsLoading(true)
+        getCommentsByArticleId(article_id)
+            .then((data) => {
+                setComments(data.comments)
+            })
+            .then(() => {
+                
+                setIsLoading(false)
+            })
+    }, [])
+
+    return isLoading ? <p>Please wait, page is loading...</p> : (
+        <div className="comment-container">
+            <h3>Comments</h3>
+            {comments.map((comment) => {
+                return (
+                <div className="comment">
+                <div className="topline">
+                <p><em>{comment.author}</em></p>
+                <p>Votes: {comment.votes}</p>
+                </div>
+                <p>{comment.body}</p>
```

Comment:
we probably want to see when the user posted the comment as well, in a human readable format